File: src/main/resources/webapp/webapp/src/app/shared/news.ts
export class News {


  constructor(public id: String,
              public _header: String,
              public _paragraph: String) {
  }


  public set header(value: String) {
    this._header = value;
  }

  public set paragraph(value: String) {
    this._paragraph = value;
  }


  public get header(): String {
    return this._header;
  }

  public get paragraph(): String {
    return this._paragraph;
  }
}

File: src/main/java/com/workoffice/entity/News.java
package com.workoffice.entity;


import javax.persistence.*;

public class News {

    @Id
    private String id;
    private String header;
    private String paragraph;

    public News() {
    }

    public News(String header, String paragraph) {
        this.header = header;
        this.paragraph = paragraph;
    }

    public String getId() {
        return id;
    }

    public String getHeader() {
        return header;
    }

    public String getParagraph() {
        return paragraph;
    }

    @Override
    public String toString() {
        return "News{" +
                "id=" + id +
                ", header='" + header + '\'' +
                ", paragraph='" + paragraph + '\'' +
                '}';
    }
}
